src/ampli/index.ts

```diff
@@ -53,27 +53,27 @@ export interface IdentifyProperties {
   /**
```

Comment:
What's all these changes?

Reply:
I pulled the latest tracking plan using `ampli pull`